README.md

```diff
@@ -126,3 +126,6 @@ We recommend your smartphone be connected to the same WiFi network as your compu
 
 - [ ] https://reactnative.dev/docs/tutorial
 - [ ] https://reactnative.dev/docs/getting-started
+
+
+
```

Comment:
Looks great, but could you remove these blank lines? Let's not have any diff for the root README.md file.

```suggestion
```

Reply:
Definitely, ok I deleted those lines.